### PR TITLE
enhance: Add cgo call metrics for load/write API

### PR DESCRIPTION
**Pull request by congqixia**
Cgo API cost is not observerable since not metrics is related to them. This PR add metrics for some sync cgo call related to load & write

### CODE REVIEW

**Comment by congqixia** [comment]
rerun cpp-unit-test

**Comment by congqixia** [comment]
rerun go-sdk

**Comment by congqixia** [comment]
rerun go-sdk

File: internal/querynodev2/segments/segment.go [hunk 1]
```diff
@@ -747,6 +748,14 @@ func (s *LocalSegment) Insert(ctx context.Context, rowIDs []int64, timestamps []
 	var status C.CStatus
 
 	GetDynamicPool().Submit(func() (any, error) {
+		start := time.Now()
+		defer func() {
+			metrics.QueryNodeCGOCallLatency.WithLabelValues(
+				fmt.Sprint(paramtable.GetNodeID()),
+				"Delete",
```
Comment (weiliu1031) [inline internal/querynodev2/segments/segment.go:755]: pass wrong param, should be Insert?

File: internal/querynodev2/segments/segment.go [hunk 2]
```diff
@@ -1055,6 +1088,14 @@ func (s *LocalSegment) LoadDeltaData(ctx context.Context, deltaData *storage.Del
...
 	var status C.CStatus
 	GetDynamicPool().Submit(func() (any, error) {
+		start := time.Now()
+		defer func() {
+			metrics.QueryNodeCGOCallLatency.WithLabelValues(
+				fmt.Sprint(paramtable.GetNodeID()),
+				"LoadFieldData",
+				"Sync",
+			).Observe(float64(time.Since(start).Milliseconds()))
+		}()
```
Comment (weiliu1031) [inline internal/querynodev2/segments/segment.go:1098]: ditto

File: internal/querynodev2/segments/segment.go (hunk 1 above)
Comment (congqixia) [reply internal/querynodev2/segments/segment.go]: fixed

File: internal/querynodev2/segments/segment.go (hunk 2 above)
Comment (congqixia) [reply internal/querynodev2/segments/segment.go:1098]: fixed

**Comment by congqixia** [comment]
rerun cpp-unit-test

**Comment by congqixia** [comment]
rerun cpp-unit-test

**Comment by congqixia** [comment]
rerun cpp-unit-test

**Pull request merged**